### PR TITLE
additional reformating

### PR DESCRIPTION
**Pull request by timse201**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Or you can exclude your particular custom post type from the confirmation dialog
 );</pre>
 
 ### Is there any way to change the default dialogue message into something else? ###
-The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme’s functions.php:
+The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme's functions.php:
```
Comment (glueckpress) [inline README.md]: We used an apostrophe (`’`) intentionally there. Any specific reason you would want to change it?

Comment (swissspidy) [inline README.md]: @glueckpress: Well, that's a closing single quote, not an apostrophe.

Comment (glueckpress) [inline README.md]: Yes, a [typographic apostrophe](http://practicaltypography.com/apostrophes.html).
Never mind, it’s an old discussion. :)

Comment (timse201) [inline README.md]: wouldn't make it without the other change ;)

**Pull request merged**